pkg/apis/infra/v1alpha1/machine.go

```diff
@@ -43,33 +42,29 @@ type CustomMachine struct {
 
 // CustomMachineSpec defines kubernetes cluster master and nodes.
 type CustomMachineSpec struct {
-	Master []Machine `json:"master,omitempty"`
-	Nodes  []Machine `json:"node,omitempty"`
+	Masters []Machine `json:"masters,omitempty"`
```

Comment:
This is a breaking change

Reply:
Or we could change them all to singular

Reply:
Current 'Master ' and 'Nodes  ' look strange because of inconsistent singular and plural forms."